test/programs/RelationalPrograms.jl

```diff
@@ -49,6 +49,12 @@ d = RelationDiagram(0)
 add_box!(d, 0, name=:A)
 @test parsed == d
 
+sird_uwd = @relation () where (S::Pop, I::Pop, R::Pop, D::Pop) begin
+  infect(S,I,I,I) # inf
+  disease(I,R) # recover
+  disease(I,D) # die
+end
+
```

Comment:
Is this supposed to be here? I don't see any tests of the constructed diagram and it doesn't seem relevant to the PR.

Reply:
The test here is just making sure that this doesn't error. 

In the course of making this PR I think there was a change which made constructing a UWD with no outer ports fail, apparently that problem has been resolved though.